lib/osf-components/addon/components/search-page/component.ts

```diff
@@ -18,7 +18,7 @@ import uniqueId from 'ember-osf-web/utils/unique-id';
 
 interface ResourceTypeOption {
     display: string;
-    value: string;
+    value: string | null;
```

Comment:
I think with this change, `@tracked resourceType: string;` should be updated to `@tracked resourceType?: string;`